### PR TITLE
Appearance attribute

### PR DESCRIPTION
**Pull request by emord**
Displaying the appearance attribute and removing the confirmation checkbox. (will run this before meilz before we merge this)

No specific tests as it doesn't look like we have specific tests for other Itext items.

@millerdev

### CODE REVIEW

**Comment by emord** [comment]
@millerdev Ameilia signed off on this

**Comment by millerdev** [comment]
Looks like labels no longer default to "minimal" appearance.

Default before:
```xml
<trigger ref="/data/question1" appearance="minimal">
  <label ref="jr:itext('question1-label')" />
</trigger>
```
Default now:
```xml
<trigger ref="/data/question1">
  <label ref="jr:itext('question1-label')" />
</trigger>
```

I suggest adding a test that starts with a blank form, adds a label, and verifies that the label (`trigger` tag) has `appearance="minimal"` in the output XML.

**Comment by emord** [comment]
@millerdev Turns out that I had the logic backwards in the tests anyways (They failed once I made the change)

File: src/javaRosa.js [hunk 1]
```diff
@@ -1895,6 +1895,18 @@ define([
...
+                widget: function (mug, options) {
+                    return itextLabelBlock(mug, $.extend(options, {
+                        itextType: "appearance",
+                        getItextByMug: function (mug) {
+                            return mug.p.appearance;
+                        },
+                        displayName: "Appearance"
+                    }));
+                }
+            };
```
Comment (millerdev) [inline src/javaRosa.js]: Help me understand this. How is appearance related to javaRosa/itext?

File: tests/expressionEditor.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    chai,
+    $,
+    util,
+    jr,
+    vellum_util,
+    TEXT_WITH_CONSTRAINT_XML
+) {
+    var assert = chai.assert;
+
+    describe("The javaRosa plugin with multiple languages", function () {
```
Comment (millerdev) [inline tests/expressionEditor.js]: Either the name or the location of this test are incorrect. Out of curiosity, is it testing anything related to appearance?

Comment (emord) [inline tests/expressionEditor.js]: That was accidentally added from my untracked files. Removed

File: src/javaRosa.js (hunk 1 above)
Comment (emord) [inline src/javaRosa.js]: Mistake by me. Removed

File: src/javaRosa.js [hunk 3]
```diff
@@ -1976,6 +1976,7 @@ define([
             );
 
             ret = ret.concat([
+                'appearance',
```
Comment (millerdev) [inline src/javaRosa.js]: Curious why this is added here and not in `getAdvancedProperties` in core.js? Appearance seems unrelated to JavaRosa (itext).

Comment (emord) [inline src/javaRosa.js]: Yeah. I think that was in my other addition to javaRosa. Changed it over to core. Should be no more changes to javaRosa here
